### DISCUSSION TITLE
Hello from odoo-lsp!

### DISCUSSION THREAD

**Discussion by Desdaemon** [odoo/odoo-ls#81] [General]
Hi, maintainer of odoo-lsp here! This looks like a very interesting project to me. May I ask what is the scope of this project and how does it compare to previous similar solutions available for PyCharm and/or VSCode? Also, is this project the official one from Odoo? I'd love to be able to exchange information and might contribute to this implementation in the future as well.

**Comment by fda-odoo** [comment]
Hello ! You're welcome :)
The scope of the project is to create extensions for main IDEs that will provide everything a dev could need that is not provided by default due to the Odoo code structure. Actually we are focused on the Language Server part, and we want it to provide all the features we can with it: autocompletion of course, hover, gotodefinition, diagnostics, signature help, etc...
How does it compare? I would say we are actually behind some good existing extensions as we only published our first beta version at OXPs, two weeks ago. This first version provide basic features, but help us a lot to improve stability and to gather needs and reaction from users to build an adapted tool for everyone. As soon as the project becomes stable, we will focus more on improving and fixing the incoherent results the tool can give now.
This is indeed an official Odoo project.
You're welcome if you want to participate, this github is opensource for this reason. We are always open to any PR, issue or discussion on this repository ;)
